volume-providers/vpc/vpcclient/models/volume_attachment.go

```diff
@@ -77,5 +84,20 @@ func (va *VolumeAttachment) ToVolumeAttachmentResponse() *provider.VolumeAttachm
 	if va.InstanceID != nil {
 		varp.InstanceID = *va.InstanceID
 	}
+
+	//Set DevicePath
+	if va.Device != nil && va.Device.ID != "" {
+		devicepath := va.Device.ID
+		generation := "gc" //default
```

Comment:
can you remove hardcoding at multiple places instead created const variable

Reply:
done